Chatkit.xcodeproj/project.pbxproj

```diff
@@ -2027,6 +1935,47 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		22B2D2A023E2F9020085D79C /* XCRemoteSwiftPackageReference "pusher-platform-swift" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/pusher/pusher-platform-swift.git";
+			requirement = {
+				branch = v1;
+				kind = branch;
+			};
+		};
+		22B2D2BF23E3004B0085D79C /* XCRemoteSwiftPackageReference "OHHTTPStubs" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/AliSoftware/OHHTTPStubs.git";
+			requirement = {
+				kind = upToNextMajorVersion;
+				minimumVersion = 9.0.0;
```

Comment:
Should `pusher-nofications-swift` be in here?  Perhaps we dropped it or perhaps its not SPM compatible?

Reply:
`PushNotifications` is compatible with SPM. I assumed that we will add this package when we will be implementing push notifications, but I can add now if you want.

`PushNotifications` is already a dependency defined by `Package.swift`.

Reply:
👍 lets not add it until we need it